nmrfx-core/src/main/java/org/nmrfx/chemistry/AtomEnergyProp.java

```diff
@@ -184,43 +182,43 @@ public static void makeIrpMap(String fileName) throws FileNotFoundException, IOE
             bf = new BufferedReader(new FileReader(fileName));
         }
 
-        lineReader = new LineNumberReader(bf);
         torsionMap.clear();
         List<List<double[]>> torsionMasterList = new ArrayList<>();
         List<double[]> torsionList;
         int nVals = 1;
         String lastKey = "";
-        while (true) {
-            String line = lineReader.readLine();
-            if (line != null) {
-                line = line.trim();
-                Matcher matcher = dihPattern.matcher(line);
-                if (matcher.matches()) {
-                    String key = getAtomKey(matcher, 4);
-                    int divider = Integer.parseInt(matcher.group(5));
-                    double barrier = Double.parseDouble(matcher.group(6));
-                    double phase = Double.parseDouble(matcher.group(7)) * Math.PI / 180.0;
-                    phase = Util.reduceAngle(phase);
-                    double periodicity = Math.abs(Double.parseDouble(matcher.group(8)));
-
-                    if (!torsionMap.containsKey(key)) {
-                        torsionList = new ArrayList<>();
-                        torsionMasterList.add(torsionList);
-                        torsionMap.put(key, torsionMasterList.size() - 1);
-                    } else {
-                        int index = torsionMap.get(key);
-                        torsionList = torsionMasterList.get(index);
-                        if (!key.equals(lastKey)) {
-                            torsionList.clear();
+        try (LineNumberReader lineReader = new LineNumberReader(bf)) {
+            while (true) {
+                String line = lineReader.readLine();
+                if (line != null) {
+                    line = line.trim();
```

Comment:
Same pattern.

Reply:
removed LineReader and replaced true with buffered reader readLine()